test/utils.ts

```diff
@@ -44,23 +54,42 @@ export async function sleep(milliseconds: number): Promise<void> {
   return new Promise(resolve => setTimeout(resolve, milliseconds))
 }
 
+/**
+ * Gets usable batch id
+ */
+export async function getUsableBatch(beeDebug?: BeeDebug): Promise<BatchId> {
+  beeDebug = beeDebug ? beeDebug : new BeeDebug(beeDebugUrl())
+  const allBatch = await beeDebug.getAllPostageBatch()
+
+  const result = allBatch.find(item => item.usable)
+
+  if (!result) {
+    throw new Error('Usable batch not found')
+  }
+
+  return result.batchID
+}
+
 /**
  * Creates and awaits for a usable batch
  */
-export async function createUsableBatch(): Promise<void> {
+export async function createUsableBatch(): Promise<BatchId> {
+  const beeDebug = new BeeDebug(beeDebugUrl())
+
   if (await isUsableBatchExists()) {
-    return
+    return getUsableBatch(beeDebug)
   }
 
-  const beeDebug = new BeeDebug(beeDebugUrl())
   await beeDebug.createPostageBatch('10000000', 24)
```

Comment:
print out the created batch with the system environment value within which this value will be saved.